ads.md

```diff
@@ -0,0 +1,131 @@
+| | PUT    | accounts/:account_id/cards/app_download/:card_id |
+| | DELETE | accounts/:account_id/cards/app_download/:card_id |
+||| *Mobile App (Image) Cards* |
+| | GET    | accounts/:account_id/cards/image_app_download |
+| | GET    | accounts/:account_id/cards/image_app_download/:card_id |
+| | POST   | accounts/:account_id/cards/image_app_download |
+| | PUT    | accounts/:account_id/cards/image_app_download/:card_id |
+| | DELETE | accounts/:account_id/cards/image_app_download/:card_id |
+||| *Lead Gen Cards* |
+| | GET    | accounts/:account_id/cards/lead_gen |
+| | GET    | accounts/:account_id/cards/lead_gen/:card_id |
+| | POST   | accounts/:account_id/cards/lead_gen |
+| | PUT    | accounts/:account_id/cards/lead_gen/:card_id |
+| | DELETE | accounts/:account_id/cards/lead_gen/:card_id |
+||| *Website Cards* |
+| | GET    | accounts/:account_id/cards/website |
+| | GET    | accounts/:account_id/cards/website/:card_id |
+| | POST   | accounts/:account_id/cards/website |
+| | PUT    | accounts/:account_id/cards/website/:card_id |
+| | DELETE | accounts/:account_id/cards/website/:card_id |
+||| *App Events* |
+| | GET    | accounts/:account_id/app_event_provider_configurations |
+| | GET    | accounts/:account_id/app_event_provider_configurations/:id |
+| | GET    | accounts/:account_id/app_event_tags |
+| | GET    | accounts/:account_id/app_event_tags/:id |
+||| *Web Events* |
+| | GET    | accounts/:account_id/web_event_tags |
+| | GET    | accounts/:account_id/web_event_tags/:web_event_tag_id |
+| | POST   | accounts/:account_id/web_event_tags |
+| | PUT    | accounts/:account_id/web_event_tags/:web_event_tag_id |
+| | DELETE | accounts/:account_id/web_event_tags/:web_event_tag_id |
+||| *Scoped Timeline* |
+| | GET    | accounts/:account_id/scoped_timeline |
+||| *Bidding Rules* |
+| | GET    | bidding_rules |
+||| *Card Stats* |
+| | GET    | stats/accounts/:account_id/cards/lead_gen/:card_id |
+| | GET    | stats/accounts/:account_id/cards/website/:card_id |
+| | GET    | stats/accounts/:account_id/cards/app_download/:card_id |
+| | GET    | stats/accounts/:account_id/cards/image_app_download/:card_id |
+||| *Campaign Stats* |
+| | GET    | stats/accounts/:account_id/campaigns/:id |
+| | GET    | stats/accounts/:account_id/campaigns |
+||| *Funding Instrument Stats* |
+| | GET    | stats/accounts/:account_id/funding_instruments/:id |
+| | GET    | stats/accounts/:account_id/funding_instruments |
+||| *Line Item Stats* |
+| | GET    | stats/accounts/:account_id/line_items/:id |
+| | GET    | stats/accounts/:account_id/line_items |
+||| *Promoted Account Stats* |
+| | GET    | stats/accounts/:account_id/promoted_accounts/:id |
+| | GET    | stats/accounts/:account_id/promoted_accounts |
+||| *Promoted Tweets Stats* |
+| | GET    | stats/accounts/:account_id/promoted_tweets/:id |
+| | GET    | stats/accounts/:account_id/promoted_tweets |
+||| *Account Stats* |
+| | GET    | stats/accounts/:account_id |
```

Comment:
Does it make sense to move this file out of source control and into [a GitHub Task List](https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments) on this pull request? I think that will make it easier to track and update progress.